distributor-api-v1/operations.md

```diff
@@ -517,10 +506,36 @@ If the hotel does not use any payment gateway, the value is null. If it does, th
 | `Description` | [Localized text](./operations.md#localized-text) | required | Description of the product localized into all supported languages. |
 | `ImageId` | string | optional | Unique identifier of the product’s image. |
 | `IncludedByDefault` | boolean | required | Indicates whether the product should be added to order by default. |
-| `Amounts` | array of [Amount](./operations.md#multi-currency-amount) | required | Array of amounts of the product. Only currencies that the property accepts are listed. |
+| `Pricing` | [Pricing coproduct](./operations.md#pricing-coproduct) | required | Object defining the pricing method and price values. |
 | `Charging` | string [Product charging](./operations.md#product-charging) | required | Charging of the product. |
 | `Posting` | string [Product posting](./operations.md#product-posting) | required | Posting of the product. |
 
+#### Pricing coproduct
+
+| Property | Type | Contract | Description |
+| :--- | :--- | :--- | :--- |
+| `Discriminator` | string [Pricing data discriminator](./operations.md#pricing-discriminator) | required | Determines type of value. |
+| `Value` | [Multi-currency amount](./operations.md#multi-currency-amount) / [Relative price data value](./operations.md#relative-price-data-value) | required | Structure of object depends on the [Pricing data discriminator](./operations.md#pricing-discriminator) |
+
+##### Pricing data discriminator
+
+* `Absolute` - Data specific to absolutely priced product are represented by [Multi-currency amount](./operations.md#multi-currency-amount).
+* `Relative` - Data specific to relatively priced product are represented by [Relative price data value](./operations.md#relative-price-data-value).
+
+##### Relative price data value
+
+| Property | Type | Contract | Description |
+| :--- | :--- | :--- | :--- |
+| `ProductIds` | array of string | required | Product ids used to calculate the price of the relatively priced product. |
+| `Multiplier` | number | required | Percentage of the relative price. |
+| `Target` | [Relative price target](./operations.md#relative-price-target) | required | Target defining whether the price of a product should be calculated from gross or net value of dependant products. |
+| `TaxRateCodes` | array of string | required | Tax rate codes that should be applied to the price in order to calculate gross price of the product. |
```

Comment:
I wouldn't say `to calculate gross price of product.` In gross-priced environments, gross comes first and then taxes are deducted from the gross in order to calculate net. 

Maybe just `in order to calculate the taxes of the product`?